cprover-cpp-api/regression/call_bmc.cpp

```diff
@@ -0,0 +1,37 @@
+// Author: Fotis Koutoulakis for Diffblue Ltd.
+
+// Test file to try loading a GOTO-model into memory and running a sample verification run on it.
+#include <util/exception_utils.h>
+
+#include "api.h"
+#include "options.h"
+
+#include <iostream>
+#include <vector>
+
+int main(int argc, char *argv[])
+{
+  try
+  {
+    std::cout << "Hello from API stub" << std::endl;
+
+    // Convert argv to vector of strings for initialize_goto_model
+    std::vector<std::string> arguments(argv + 1, argv + argc);
+
+    // Create API options object, to pass to initialiser of API object.
+    auto api_options = api_optionst::create().simplify(false);
```

Comment:
Why hardcoded to false

Reply:
(This is a single option for now, while we set up the scaffold for further work - we intend to expand this with way more options).